cribl_stream/README.md

```diff
@@ -0,0 +1,69 @@
+# Cribl Stream
+
+## Overview
+[Cribl Stream][1] helps you process machine data – logs, instrumentation data, application data, metrics, and so on – in real time, and deliver it to your analysis platform of choice. It allows you to:
+
+- Add context to your data, by enriching it with information from external data sources.
+- Help secure your data, by redacting, obfuscating, or encrypting sensitive fields.
+- Optimize your data, per your performance and cost requirements.
+
+Use the out-of-the-box dashboard to view the performance of the Stream with base metrics like events per second, bytes per second, input types, output types, and infrastructure metrics. Monitor reduction percentages by events or bytes, which is useful for improving search performance or licensing and infrastructure costs for the systems of analysis.
```

Comment:
To clarify - by reduction percentages, is this the amount of time/money/etc saved after data is optimized using Cribl? May be helpful to spell that out

Reply:
Updated the text in the overview to include calling that out. "Use the Datadog dashboard for Cribl Stream to monitor reduction percentages by events or bytes. This is useful for improving search performance or licensing and infrastructure costs for the systems of analysis."